l1-contracts/contracts/state-transition/chain-deps/facets/Executor.sol

```diff
@@ -423,9 +429,14 @@ contract ExecutorFacet is ZKChainBase, IExecutor {
         // Save root hash of L2 -> L1 logs tree
         s.l2LogsRootHashes[_storedBatch.batchNumber] = _storedBatch.l2LogsTreeRoot;
 
-        // Once the batch is executed, we include its message to the message root.
-        IMessageRoot messageRootContract = IBridgehub(s.bridgehub).messageRoot();
-        messageRootContract.addChainBatchRoot(s.chainId, currentBatchNumber, _storedBatch.l2LogsTreeRoot);
+        // During migration to the new protocol version, there will be a period when
```

Comment:
I would probably create a function to avoid duplication, but it is also ok like this

Reply:
agree